chart/values.schema.json

```
@@ -3764,6 +3764,19 @@
                             }
                         }
                     }
+                },
```

```suggestion
                },
                "uid": {
                    "description": "Redis run as user parameter.",
                    "type": "integer",
                    "default": 999
                },
```

Which means we also need `uid`.